extensions/amp-next-page/amp-next-page.md

```diff
@@ -24,25 +24,17 @@ limitations under the License.
 
 # amp-next-page
 
-## Behavior
-
-Given a list of pages, `amp-next-page` tries to load them after the current
-document, providing an infinite-scroll type experience.
+## Usage
 
-Pages should be specified in a JSON config loaded from a remote URL listed in
-the element `src`, or inlined as a `script` element child.
+The `<amp-next-page>` component loads content pages one after another creating an infinite scroll experience.
 
-## Version notes
+### Version notes
```

Comment:
Why is this now nested? BTW, what's the standard place to embed version info?

Reply:
Good questions:
_Why is this now nested?_ 
I figured it was better to present what the component did for new users instead of assuming they were there to upgrade to the newest version. It makes sense to me to go "this is amp-next-page and it does XYZ, also, there was two different versions, here's what you need to know."

_BTW, what's the standard place to embed version info?_
There is no standard place right now. I'm open to suggestions. I would like to not keep it a top-level heading, as I feel its relevancy fades as we move further away from the version release date.

wdyt?

Reply:
I think it doesn't fit under usage as it's not related to it's usage, but a separate topic. But I agree with your concern of fading relevancy. How about keeping it as a top level heading, but move it to the bottom? This way the information is available (and can be quickly accessed via the TOC), but distracts from the main usage.

Reply:
Sure, that sounds reasonable to me. I'l also add this to the template.